inputs-optional.tf

```diff
@@ -178,4 +178,79 @@ variable "enable_qpi" {
   type        = bool
   default     = false
   description = "Enables Query Performance Insight. Creates Log Analytics workspace and diagnostic setting needed"
+}
+
+variable "action_group_name" {
+  description = "The name of the Action Group to create."
+  type        = string
+  default     = "db_alerts_action_group_name"
+}
+
+variable "email_receivers" {
+  description = "A map of email receivers, with keys as names and values as email addresses."
+  type        = map(string)
+  default     = {}
+}
+
+variable "sms_receivers" {
+  description = "A map of SMS receivers, with keys as names and values as maps containing country code and phone number."
+  type = map(object({
+    country_code = string
+    phone_number = string
+  }))
+  default = {}
+}
+
+variable "webhook_receivers" {
+  description = "A map of webhook receivers, with keys as names and values as URLs."
+  type        = map(string)
+  default     = {}
+}
```

Comment:
Think these can all now be deleted, to avoid confusion?

Reply:
done